app/tests/README.md

```
@@ -1,30 +1,18 @@
-# Tests Documentation
+# 🧪 Tests in Docker — Command Cheat Sheet
+| Goal                                     | Docker Command                                                                             | Makefile    |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)